spec/end_to_end/end_to_end_spec.rb

```diff
@@ -58,29 +58,31 @@
     end
   end
 
-  context "when the form has a file upload" do
-    let(:file_question_text) { "Upload a file" }
-    let(:test_file) { "/tmp/temp-file.txt" }
+  unless ENV.fetch('SKIP_FILE_UPLOAD', false)
+    context "when the form has a file upload" do
```

Comment:
It's not part of your change, but I think this would be clearer as:
```suggestion
    context "when the form has a file upload question" do
```